### PR TITLE
:sparkles: alerts

### PR DESCRIPTION
**Pull request by RobbeB**

### CODE REVIEW

File: src/components/Alerts/Alert.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    removeAlert(alert.id);
+  }, [removeAlert, alert.id]);
+
+  React.useEffect(() => {
+    if (!noAutoClose) {
+      setTimeout(
+        () => {
+          removeSelf();
+        },
+        alert.type !== AlertType.Success ? 5000 : 10000,
```
Comment (THMKAE) [inline src/components/Alerts/Alert.tsx]: Maybe it's a good idea to also make this configurable.
This way in future implementations you can override the default time.

File: src/components/Alerts/AlertWrapper.ts [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+import { AlertType } from './Alerts';
+import { media, ComponentTheme } from '../../theme';
+
+const typeColors = (
+  props: ThemedStyledProps<{ index: number; type: AlertType }, ComponentTheme>,
+): { color: string; background: string } => {
+  switch (props.type) {
+    case AlertType.Danger:
+      return {
+        background: props.theme.alerts.color.dangerSecondary,
```
Comment (THMKAE) [inline src/components/Alerts/AlertWrapper.ts]: A reason why this isn't coming from a background property?
So `props.theme.alerts.background.danger`.

File: src/components/Alerts/index.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export { default as Alerts, AlertType, AlertWithIdentifier } from './Alerts';
```
Comment (THMKAE) [inline src/components/Alerts/index.ts]: Maybe we should also export the Alert as well? This way you can integrate it outside of the Alerts container. For example within some documentation (and within the stories).

Comment (RobbeB) [reply src/components/Alerts/index.ts]: I decided not to export the Alert on purpose. The only thing that would accomplish is that the user can use a single Alert outside of the AlertsContainer, which I don't see a useful usecase for. And all documentation should be derived from the Alerts component, which has all the types for the Alert

File: src/components/Alerts/AlertWrapper.ts (hunk 2 above)
Comment (RobbeB) [reply src/components/Alerts/AlertWrapper.ts]: You are right, this was a blind copy from the existing code, i'll change it

**Pull request merged**